app/MMB/src/scenes/tripOverview/tripTitle/TripTitle.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet } from '@kiwicom/mobile-shared';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+
+import TripOverviewContext, { type BookingType } from '../TripOverviewContext';
+import type { TripTitle as TripTitleType } from './__generated__/TripTitle.graphql';
+import MulticityTitle from './MulticityTitle';
+import Duration from '../../../components/header/Duration';
+
+type PropsWithContext = {|
+  ...Props,
+  +type: BookingType,
+|};
+
+const TripTitle = ({ type, data }: PropsWithContext) => {
+  return (
+    <View style={styles.row}>
+      {type === 'BookingMulticity' && <MulticityTitle data={data} />}
+      {/* TODO: Return */}
+      {/* TODO: OneWay */}
+      <Duration showIcon={false} data={data} />
```

Comment:
There should be a bigger gap here, maybe a `<Translation passThrough=" " />` will be enough.